assets/tslint.json

```diff
@@ -10,7 +10,7 @@
   "rules": {
     "interface-name": [true, "never-prefix"],
     "jsx-no-lambda": false,
-    "object-literal-sort-keys": { "options": "match-declaration-order-only" },
+    "object-literal-sort-keys": false,
```

Comment:
Are we sure we want to get rid of this? It's sort of annoying when making changes but IMO does help readability with objects that have a ton of keys.

Reply:
So in `notificationsContext.tsx` we have:

```
interface State extends ReducerState {
  dispatch: (action: Action) => void
}
```

The `ReducerState` that we're extending has two other fields, so we end up with an interface with three fields. So far so good. But when you have a `State` in the code, there is no order of the three fields that will satisfy this linter rule. I assume that's due to a bug in the linter.

We could turn it off on a case by case basis but this `State` gets used a lot--the number of `disable` and `enable` comments I was adding was ridiculous. I figured switching off the rule entirely was a lesser evil.

Reply:
Ahh, okay, now that you mention it I feel like I've run into either this or a similar linter issue before, though I forget what I did in that case. That makes sense (and I agree that switching it off case-by-case is probably worse).